cdk-infra/lib/constructs/queue/queues-construct.ts

```diff
@@ -13,6 +14,7 @@ export class AutoBuilderQueuesConstruct extends Construct {
     const jobsQueueDlq = new Queue(this, 'jobsQueueDlq');
 
     const jobsQueue = new Queue(this, 'JobsQueue', {
+      visibilityTimeout: Duration.seconds(120),
```

Comment:
This doesn't cause problems with deploying to `prd`, but creating a new stack without adjusting the `visibilityTimeout` to be greater than or equal to a consuming lambda function's `timeout` leads to the error when deploying that is mentioned in the OP. 

Basically, the `visibilityTimeout` is the period in which a message has been picked up by a consumer, and is no longer visible to other consumers. During this period, it is expected that the consumer that picked up the message will process the message and delete it. Otherwise, when the `visibilityTimeout` duration is reached, the message will now be available to other consumers if it is not deleted.

[A more detailed explanation for this can be found here.](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html)